x-pack/plugins/alerting/public/pages/maintenance_windows/components/license_prompt.tsx

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiLink, EuiPageTemplate } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+import * as i18n from '../translations';
+
+const title = <h2 data-test-subj="license-prompt-title">{i18n.UPGRADE_TO_PLATINUM}</h2>;
+const body = (
+  <FormattedMessage
+    data-test-subj="license-prompt-body"
+    defaultMessage="Maintenance windows is available when you have the {appropriateLicense}, are using a {cloud}, or are testing out a Free Trial."
```

Comment:
I suggest updating this text to read "Use a cloud deployment, start a trial, upgrade your subscription level.":
```suggestion
    defaultMessage="Use a {cloud}, start a trial, or upgrade your {appropriateLicense}."
```